src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -289,6 +289,66 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         }
     }
 
+    /**
+     * @notice Updates the stakes of all operators for each of the specified quorums in the StakeRegistry. Each quorum also
+     * has their StakeRegistry.quorumTimestamp updated. which is meant to keep track of when operators were last all updated at once. 
+     * @param operatorsPerQuorum is an array of arrays of operators to update for each quorum. Note that each nested array
+     * of operators must be sorted in ascending operatorId order to ensure that all operators in the quorum are updated
+     * @param quorumNumbers is an array of quorum numbers to update
+     * @dev This method is used to update the stakes of all operators in a quorum at once, rather than individually
+     * 
+     */
+    function updateOperatorsPerQuorum(address[][] calldata operatorsPerQuorum, bytes calldata quorumNumbers) external onlyWhenNotPaused(PAUSED_UPDATE_OPERATOR) {
+        require(
+            operatorsPerQuorum.length == quorumNumbers.length,
+            "BLSRegistryCoordinatorWithIndices.updateOperatorsPerQuorum: input length mismatch"
+        );
+        for (uint256 i = 0; i < quorumNumbers.length; ++i) {
+            uint8 quorumNumber = uint8(quorumNumbers[i]);
+            address[] memory currQuorumOperators = operatorsPerQuorum[i];
+            require(
+                currQuorumOperators.length == indexRegistry.totalOperatorsForQuorum(quorumNumber),
+                "BLSRegistryCoordinatorWithIndices.updateOperatorsPerQuorum: number of updated operators does not match quorum total"
+            );
+            bytes32 prevOperatorId = bytes32(0);
+
+            // Update stakes for each operator in this quorum
+            for (uint256 j = 0; j < currQuorumOperators.length; ++j) {
+                address operator = currQuorumOperators[j];
+                bytes32 operatorId = getOperatorId(operator);
+                uint192 currentBitmap = _currentOperatorBitmap(operatorId);
```

Comment:
Would prefer we do a similar pattern like the one in `updateOperators` that validates that the `operator` has `OperatorStatus.REGISTERED`

Reply:
Cause currently this allows you to update without being registered (i think). `_currentOperatorBitmap` returns 0 if you're not registered.

Reply:
ah i see there's a check that the number is in the bitmap. Regardless - I like the explicit check; makes it easier to surface and enforce the status invariants

Reply:
I thought about that too, but we have to ensure the operator in the array is registered for the corresponding quorum. Not just at least one quorum

Reply:
Yeah - I'm not suggesting removing specific-quorum check, just explicitly making the REGISTERED check - just so it's explicit and clear. It shouldn't be an extra storage read, since you're doing the same SLOAD to get the operatorId

Reply:
It's nbd if you feel it adds clutter.

Maybe a compromise could be to add a comment above the specific quorum check, that this enforces the REGISTERED status invariant?

Reply:
IMO the explicit REGISTERED check would be clearest, but I think the comment would be fine. really good point about us already SLOAD'ing the same slot.

Reply:
https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/0556f9276126cebd4eecde019282670365a175db
Added check similar to `updateOperators` pattern